src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -7,32 +7,54 @@ import { ServiceContextProvider } from '../contexts/ServiceContext';
 import { RequestContextProvider } from '../contexts/RequestContext';
 import GlobalStyle from '../lib/globalStyles';
 import ConsentBanner from '../containers/ConsentBanner';
+import getStatsDestination from '../contexts/RequestContext/getStatsDestination';
+import getStatsPageIdentifier from '../contexts/RequestContext/getStatsPageIdentifier';
 
-const PageWrapper = ({ bbcOrigin, children, service, isAmp }) => (
-  <Fragment>
-    <GlobalStyle />
-    <RequestContextProvider
-      platform={isAmp ? 'amp' : 'canonical'}
-      bbcOrigin={bbcOrigin}
-    >
+const PageWrapper = ({ bbcOrigin, children, id, service, isAmp }) => {
+  const env = process.env.APP_ENV;
+  console.log(
+    getStatsDestination({ isUK: false, env, service }),
+    getStatsPageIdentifier({ pageType: 'article', service, id }),
+    env,
+  );
+  return (
+    <Fragment>
+      <GlobalStyle />
       <ServiceContextProvider service={service}>
-        <Helmet>
-          <link rel="manifest" href={`/${service}/articles/manifest.json`} />
-        </Helmet>
-        <ConsentBanner />
-        <HeaderContainer />
-        {children}
-        <FooterContainer />
+        <RequestContextProvider
+          id={id}
+          statsDestination={getStatsDestination({
+            isUK: false,
```

Comment:
The function `getStatsDestination` requires the `service` value. That is why it is being run here in the `defaultPageWrapper` where `service` is defined above, instead of within the `RequestContext` file with no access to `service`. 
However, this function also needs `isUK` to be defined. This value is only available within `RequestContext`. (Currently I've just hard-coded the value to `false`, but that needs to change)

There's one option I can think to address this - to pull out `getOrigin()` from the RequestContext file and run it here
```
const { isUK, origin} = getOriginContext(bbcOrigin);
```
Then pass in `isUK` as an argument to `getStatsDestination` as well as pass in props `isUK` & `origin` to the RequestContext. 

Since this will involve refactoring the existing logic, I'd be interested in getting views from @hindsc52 & others.  

If there's another cleaner approach that I'm missing, I'd appreciate hearing that, too.

Reply:
I've carried out the changes explained above in this commit: https://github.com/bbc/simorgh/pull/1711/commits/da8925a51b9dd7e1afe7d0319024b74641f2d811

Reply:
@hindsc52 Thank you for looking at this. Okay, I'll just add unit tests to `defaultPageWrapper` so we can check the various inputs/outputs, then move the PR out of draft mode.